python/turbodbc_test/test_select_numpy.py

```diff
@@ -109,6 +121,19 @@ def test_numpy_binary_column_larger_than_batch_size(dsn, configuration):
             expected = MaskedArray([1, 2, 3, 4, 5], mask=False)
             assert_equal(results[_fix_case(configuration, 'a')], expected)
 
+@for_each_database
+def test_numpy_batch_fetch(dsn, configuration):
+    with open_cursor(configuration, rows_to_buffer=2) as cursor:
+        with query_fixture(cursor, configuration, 'INSERT INTEGER') as table_name:
+            cursor.executemany("INSERT INTO {} VALUES (?)".format(table_name),
+                               [[1], [2], [3], [4], [5]])
+            cursor.execute("SELECT a FROM {} ORDER BY a".format(table_name))
+            batches = cursor.fetchnumpybatches()
+            expected_list = [MaskedArray([1, 2], mask=False), MaskedArray([3, 4], mask=False), MaskedArray([5], mask=False)]
```

Comment:
please use line breaks to separate the batches visually:

```
expected_list = [MaskedArray([1, 2], mask=False),
                 MaskedArray([3, 4], mask=False),
                 MaskedArray([5], mask=False)]
```
Also, `expected_list` should probably be renamed to `expected_batches`